package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Airbnb Listing Clone - Description Component",
   "main": "index.js",
   "scripts": {
-    "react-dev": "webpack -d --watch",
+    "react-dev": "webpack -p -d --watch",
```

Comment:
Great that you are using the production flag for your bundles that will be deployed. I would suggest a separate `react-dev` and `react-prod` script, since the `-p` makes the bundling slower, so in a `--watch` situation you'll want it to be as fast as possible to avoid waiting while in development mode.